extension/src/Common.ts

```diff
@@ -38,3 +38,27 @@ export function formatDate(date = new Date()): string {
 
   return `${year}-${month}-${day}`;
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function sortObjByKey(value: any): any {
+  return typeof value === "object"
+    ? Array.isArray(value)
+      ? value.map(sortObjByKey)
+      : Object.keys(value)
+          .sort()
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          .reduce((o: any, key) => {
+            const v = value[key];
```

Comment:
I would have skipped this declaration but I guess this is you setting up for debugging :wink:

Reply:
actually it's just copy&paste from SO. But I guess that someone else then me likes to declare variables/consts 😅